assignments/arrays.js

```diff
@@ -63,32 +63,59 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+//console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+let car33;
+for(let i = 0; i < inventory.length; i++){
+  if(inventory[i].id === 33){
+    car33 = inventory[i];
+  }
+}
+console.log(`Car 33 is a ${car33.car_year} ${car33.car_make} ${car33.car_model}` );
+
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+let lastCar = inventory [inventory.length-1];
+console.log('The last car in the inventory is ${lastCar.car_make} ${lastCar.car_model}');
```

Comment:
Confident that this is merely a typo, but make sure that you're using backticks (not single quotes) for template string literals (where you're trying to insert variable values into your string inside `console.log`).